### ISSUE TITLE
Not compatible with Gradle 7 and React Native 0.65.1 on Android

### ISSUE THREAD

**Issue by lukebrandonfarrell** [iamolegga/react-native-launch-arguments#32]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-launch-arguments@2.0.1` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-launch-arguments/android/build.gradle b/node_modules/react-native-launch-arguments/android/build.gradle
index ff84199..3fdbc79 100644
--- a/node_modules/react-native-launch-arguments/android/build.gradle
+++ b/node_modules/react-native-launch-arguments/android/build.gradle
@@ -20,7 +20,7 @@ def safeExtGet(prop, fallback) {
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 buildscript {
     // The Android Gradle plugin is only required when opening the android folder stand-alone.
@@ -39,7 +39,7 @@ buildscript {
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 android {
     compileSdkVersion safeExtGet('compileSdkVersion', DEFAULT_COMPILE_SDK_VERSION)
@@ -140,10 +140,9 @@ afterEvaluate { project ->
 
     task installArchives(type: Upload) {
         configuration = configurations.archives
-        repositories.mavenDeployer {
+        repositories.maven {
             // Deploy to react-native-event-bridge/maven, ready to publish to npm
-            repository url: "file://${projectDir}/../android/maven"
-            configureReactNativePom pom
+            url = uri("file://${projectDir}/../android/maven")
         }
     }
 }
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by iamolegga** [comment]
Thanks, could you open a PR with this? I would be happy to merge it

**Comment by lukebrandonfarrell** [comment]
@iamolegga Hmm I'm not able to, when I pull the repository, the Gradle file is different to what I get from NPM. Looking further, it seems as if this library isn't up to date on NPM.

**Comment by HannahCarney** [comment]
I need this also

**Comment by iamolegga** [comment]
I believe this is fixed in #34 

Please check version **3.1.0**
